application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -27,6 +32,8 @@
  */
 public final class TagsCommand extends SlashCommandAdapter {
     private final TagSystem tagSystem;
+    private static final Logger logger = LoggerFactory.getLogger(TagsCommand.class);
+    private static final int MAX_TAGS_THRESHOLD_WARNING = 200;
```

Comment:
NIT: usually static stuff comes before non static stuff. if you want, you could reorder this, i.e. `tagSystem` below the other two. ur choice though.

Reply:
fixed